src/js/utils/reactiveRender.js

```diff
@@ -0,0 +1,11 @@
+const reactiveRenderList = [];
+
+function registeReactiveRender(render) {
+  reactiveRenderList.push(render);
+}
+
+function reactiveRender() {
+  reactiveRenderList.forEach(renderFunction => renderFunction());
+}
+
+export { reactiveRenderList, registeReactiveRender, reactiveRender };
```

Comment:
reactiveRenderList까지 export 할 필요가 있을까요?
```suggestion
const reactiveRenderList = [];

export function registeReactiveRender(render) {
  reactiveRenderList.push(render);
}

export function reactiveRender() {
  reactiveRenderList.forEach(renderFunction => renderFunction());
}
```
이런식으로만 표현해도 좋을 것 같아요!

그리고 네이밍은 render에 종속적이지만, 실제 코드를 살펴보면 render와 전혀 상관이 없네요!

Reply:
reactiveRenderList는 export할 필요없습니다!! 제거 하였습니다!
하지만 코드는 render에 종속적이지 않지만 의미적으론 등록된 render function을 돌려주는 역할을 하게 됩니다 이럴 경우는 어떤식으로 작성하면 좋을까요?!
(먼가...제가 구조를 처음부터 잘못잡아서 이렇게된것 같기도합니다! ㅠ)

Reply:
이런 것들을 보통 Observer Pattern 이라고 표현한답니다 ㅎㅎ

- 구독 (subscribe): 콜백 함수 등록
- 알림 (notify): 등록된 콜백 함수들 실행